windows/src/engine/keyman32/hotkeys.cpp

```diff
@@ -26,9 +26,7 @@
    // I5136
 #include "pch.h"
 
-#ifdef _WIN64
-#error hotkeys.cpp is not required for win64.
-#endif
+#ifndef _WIN64
```

Comment:
Swapping the condition around, because the .cpp files are now included with all archs, and it is easier to track in the .cpp than having conditional exclusions in the .vcxproj